src/legacy/core_plugins/kibana/public/management/index.js

```diff
@@ -108,9 +110,16 @@ export function updateSidebar(items, id) {
     return;
   }
 
+  const npSections = npStart.plugins.management.sections.getAllSections();
```

Comment:
I don't think its necessary to prefix with np.
I haven't seen this pattern elsewhere.